lib/find_non_repeated_letter.rb

```diff
@@ -9,6 +9,28 @@
 
 
 def find_first_non_repeated_letter(str)
-  #write your method here
-  return true
+  
+  array = str.chars
+  
+  final = array.length - 1
+  
+  answer = ""
+  
+  array.each_index do |i|
+    if i == 0 && array[i] != array[(i+1)]
```

Comment:
Do you _need_ to be in a loop to do this check? Is there ever a case where this conditional would run where `i` would not be `0`? If not, then this isn't really an "iterative" operation–it's "static". You could just as easily run the check outside of the loop and hardcode in the array's position.